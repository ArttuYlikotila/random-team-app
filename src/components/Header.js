export default function Header() {
  return (
    <div className='pt-3 pb-3 mb-5 shadow'>
      <h1>Random Team App</h1>
    </div>
  );
}