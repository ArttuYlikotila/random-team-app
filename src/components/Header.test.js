import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Header from './Header';

describe('Header component', () => {
  test('Renders correctly', () => {
    render(<Header />);

    expect(screen.getByRole('heading')).toMatchSnapshot();
    expect(screen.getByRole('heading')).toBeInTheDocument();
  })
})