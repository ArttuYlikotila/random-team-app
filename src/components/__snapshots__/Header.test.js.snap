// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Renders correctly 1`] = `
<h1>
  Random Team App
</h1>
`;
